specs/fjord/deposits.md

```diff
@@ -0,0 +1,93 @@
+# Deposits
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [L1 Attributes Deposited Transaction](#l1-attributes-deposited-transaction)
+  - [L1 Attributes Deposited Transaction Calldata](#l1-attributes-deposited-transaction-calldata)
+    - [L1 Attributes - Fjord](#l1-attributes---fjord)
+- [Special Accounts on L2](#special-accounts-on-l2)
+  - [L1 Attributes Predeployed Contract](#l1-attributes-predeployed-contract)
+    - [Fjord L1Block upgrade](#fjord-l1block-upgrade)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## L1 Attributes Deposited Transaction
+
+### L1 Attributes Deposited Transaction Calldata
+
+#### L1 Attributes - Fjord
+
+On the Fjord activation block, and if Fjord is not activated at Genesis,
+the L1 Attributes Transaction includes a call to `setL1BlockValuesEcotone()`
+because the L1 Attributes transaction precedes the [Fjord Upgrade Transactions][fjord-upgrade-txs],
+meaning that `setL1BlockValuesFjord` is not guaranteed to exist yet.
+
+Every subsequent L1 Attributes transaction should include a call to the `setL1BlockValuesFjord()` function.
+The input args are no longer ABI encoded function parameters,
+but are instead packed into 5 32-byte aligned segments and one 12-byte segment (starting after the function selector).
+Each signed and unsigned integer argument is encoded as big-endian using a number of bytes corresponding to the
+underlying type. The overall calldata layout is as follows:
+
+[fjord-upgrade-txs]: derivation.md#fjord
+
```

Comment:
This is ignoring the diff based L1 attributes design. How do you feel about using the diff based design?

Reply:
As discussed in the meeting, we simplified and removed the L1 attributes altogther